### PR TITLE
Refactor command to constants

### PR DESCRIPTION
**Pull request by vkhramtsov**
This PR:
- [x] Covered by tests

### CODE REVIEW

File: src/Command/RunCommand.php [hunk 1]
```diff
@@ -159,73 +228,76 @@ protected function configure(): void
                 Container::DEFAULT_FORMATTER_NAME
             )
             ->addOption(
-                'git-diff-filter',
+                self::OPTION_GIT_DIFF_FILTER,
                 null,
                 InputOption::VALUE_REQUIRED,
                 'Filter files to mutate git `--diff-filter` options. A - only for added files, AM - for added and modified.',
```
Comment (sanmai) [inline src/Command/RunCommand.php:234]: `--diff-filter`?

Comment (vkhramtsov) [reply src/Command/RunCommand.php:234]: I think that it is not applicable here because we have `diff-filter` only once in this command

Comment (vkhramtsov) [reply src/Command/RunCommand.php:234]: I mean in this file

**Review by sanmai — COMMENTED** [review]
On one hand this is something I'd do if I'd have to make a command from scratch. On other hand I can't help but notice that the code became less readable after this change.

**Comment by vkhramtsov** [comment]
At the same time this change able to help with decreasing possibility of error. Anyway, it is just proposal

**Review by maks-rafalko — APPROVED** [review]
I don't think we will often read this code, so as for me - having constants probably better than having a possible error. But that's just my opinion

**Comment by sanmai** [comment]
Fair enough.

**Comment by sanmai** [comment]
Fair enough.

Thank you @vkhramtsov

**Pull request merged**